app/models/Orders.js

```diff
@@ -0,0 +1,128 @@
+const knex = require('../../db');
+const Joi = require('joi');
+
+class OrdersModel {
+    constructor() {
```

Comment:
pass data throw constructor parameters and in `validateId` and `validate` use object properties, not params